alsaaudio.c

```diff
@@ -639,6 +636,201 @@ alsapcm_dumpinfo(alsapcm_t *self, PyObject *args)
 	return Py_None;
 }
 
+
+static PyObject *
+alsapcm_info(alsapcm_t *self, PyObject *args)
+{
+	PyObject *info = PyDict_New();
+	PyObject *pname;
+	PyObject *value;
+
+	unsigned int val,val2;
+	snd_pcm_format_t fmt;
+	int dir;
+	snd_pcm_uframes_t frames;
+	snd_pcm_hw_params_t *hwparams;
+	snd_pcm_hw_params_alloca(&hwparams);
+	snd_pcm_hw_params_current(self->handle,hwparams);
+
+	if (!PyArg_ParseTuple(args,":dumpinfo"))
+		return NULL;
+
+	if (!self->handle) {
+		PyErr_SetString(ALSAAudioError, "PCM device is closed");
+		return NULL;
+	}
+
+	printf("PCM handle name = '%s'\n", snd_pcm_name(self->handle));
```

Comment:
That `printf` can/should be removed here.

Reply:
Done.